.github/workflows/pr-open.yml

```diff
@@ -312,6 +313,8 @@ jobs:
         with:
           context: .
           push: true
+          build-args: |
+            KEYCLOAK_SERVER_REALM=${{ secrets.KEYCLOAK_SERVER_REALM }}
```

Comment:
Why are we passing down the realm during build time? 
This will "hardcode" one specific realm on the image, making it part of the image itself. For something that will/could change depending on the environment, it is better to use an environment variable or even an external file configuration.

Reply:
Yeah. That's an excellent question! We shouldn't hard code. I found an issue related to environment variables. Give me some minutes to explore and dig about it.

Reply:
Fixed. It was an issue related to default env vars in the properties file. Thanks for pointing out.